public/game/views/game-view.coffee

```diff
@@ -32,13 +34,14 @@ GameView = (context) ->
 									nxt.Text 'Banned characters'
 									OccurrenceIndicatorView context.occurrenceIndicator
 
-
-				nxt.Element 'input',
-					nxt.Class 'controls-selector-input'
-					nxt.ValueBinding context.selector
-					nxt.Attr 'placeholder', 'Enter your selector here...'
-					# nxt.Binding context.match, (match) ->
-					# 	nxt.Attr 'disabled' if match?.result is SelectorMatchResult.POSITIVE
+				nxt.Binding context.round_phase, (phase) ->
+					if phase is RoundPhase.IN_PROGRESS
+						nxt.Element 'input',
+							nxt.Class 'controls-selector-input'
+							nxt.Event 'input', context.selector, (event) -> event.target.value
+							nxt.Attr 'placeholder', 'Enter your selector here...'
+							# nxt.Binding context.match, (match) ->
```

Comment:
Not really, the match cell should also be cleared. Let's do it separately if you don't mind.